tests/vmi_cloudinit_test.go

```diff
@@ -668,3 +630,8 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 	})
 })
+
+func verifyUserDataVMI(vmi *v1.VirtualMachineInstance, timeout time.Duration) {
+	By("Checking whether the user-data script had created the file")
+	Expect(console.RunCommand(vmi, fmt.Sprintf("cat /%s\n", expectedUserData), timeout)).To(Succeed())
```

Comment:
I do not understand why you need a helper function now and I dislike the inner assertion (vs returning an error). But if you still keep it, maybe it is worth using the assertion with an offset so in case of failure we will see the right stack/line ref.

Reply:
Agree, it is redundant.